### ISSUE TITLE
JVM arguments after -classpath cause failure

### ISSUE THREAD

**Issue by pron** [cloudius-systems/osv#527]
The bug is on [this line](https://github.com/cloudius-systems/osv/blob/master/java/java.cc#L138): when `java_main` encounters an argument not starting with `-` -- like the classpath following `-classpath` -- it exits the loop and passes everything else to `RunJava`.

**Comment by wkozaczuk** [comment]
Given we are close to support running unmodified JREs and little appetite in maintaining java.cc I suggest we close this issue

**Comment by nyh** [comment]
I don't think we should rush to fix it, but let's not close it if it's still a correct description of a problem. We can close it if the problem no longer exists, or we delete the separate java.cc.
